### ISSUE TITLE
[a11y] close datepicker and maintain focus on button

### ISSUE THREAD

**Issue by Etat-Plateforme** [angular-ui/bootstrap#4253]
According to http://www.w3.org/TR/wai-aria-practices/#datepicker:
_escape_ - in the case of a popup date picker - closes the widget without any action.
It shoudn’t change focus to input but must stay on button.

**Comment by wesleycho** [comment]
That link does not seem to suggest what is being requested here, and [this accessibility example](http://oaa-accessibility.org/example/15/) seems to suggest that the current behavior is indeed the correct one.

I am inclined to think that this is not a bug and this should be closed, but I would like verification since I'm not quite familiar with accessibility support.

**Comment by icfantv** [comment]
Closing in favor of #4772.
